Data/models/Domain.swift

```diff
@@ -31,13 +37,36 @@ public final class Domain: NSManagedObject, CRUD {
     public override func awakeFromInsert() {
         super.awakeFromInsert()
     }
+    
+    // Returns `url` but switches the scheme from `http` <-> `https`
+    private func domainForInverseHttpScheme() -> Domain? {
+        // Must be an `http` related domain
```

Comment:
Not really an issue but I got confused for a second with this logic 😅, maybe instead of `Must be an `http` related domain`, could say `Scheme must be `http` or 'https'`?